docs/breaking-changes.asciidoc

```diff
@@ -3,24 +3,30 @@
 
 ### E_USER_DEPRECATED notice when using deprecated parameters
 
-Starting from elasticsearch-php 7.4.0, we generate a PHP [E_USER_DEPRECATED](https://www.php.net/manual/en/errorfunc.constants.php)
-notice every time you use a deprecated parameters for Elasticsearch.
-We decided to add this notice to facilitate the code refactoring with the
-new API specification of Elasticsearch (e.g. the usage of typeless APIs, see below).
+Starting from elasticsearch-php 7.4.0, we generate a PHP 
+https://www.php.net/manual/en/errorfunc.constants.php[E_USER_DEPRECATED] notice 
+every time you use a deprecated parameters for {es}. We decided to add this 
+notice to facilitate the code refactoring with the new API specification of {es} 
+(for example the usage of typeless APIs, see below).
```

Comment:
I forgot to mention here that we silent the `E_USER_DEPRECATED` warning in [7.4.1](https://github.com/elastic/elasticsearch-php/releases/tag/v7.4.1) (using the `@` operator)  due to BC break issues in other libraries using elasticsearch-php. The full story can be found [here](https://github.com/elastic/elasticsearch-php/pull/973).

Reply:
@ezimuel 
Thank you for the review and for the note. I added the story to the `E_USER_DEPRECATED` section in e8d582d. Please let me know if you want to change/add anything.